### PR TITLE
fix: move docker mongodb service to a distinct file in dev mode

### PR DESCRIPTION
**Pull request by raphael0202**
To avoid conflicts with Product Opener docker mongo instance in dev mode

### CODE REVIEW

File: docker/mongodb.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: "3.9"
+
+services:
+  mongodb:
+    image: mongo:4.4
+    command: mongod --wiredTigerCacheSizeGB 1.5
+    ports:
+      - "127.0.0.1:${MONGO_EXPOSE_PORT:-27017}:27017"
```
Comment (alexgarel) [inline docker/mongodb.yml:8]: sorry, I didn't spot it before, but we need a volume here ! If not mongo data won't be persistent…

Comment (alexgarel) [reply docker/mongodb.yml:8]:
```suggestion
services:
  mongodb:
    image: mongo:4.4
    command: mongod --wiredTigerCacheSizeGB 1.5
    ports:
      - "127.0.0.1:${MONGO_EXPOSE_PORT:-27017}:27017"
    volumes:
      - mongodb_data:/data/db
      
volumes:
  mongodb_data:
```

Comment (raphael0202) [reply docker/mongodb.yml:8]: Indeed!

**Review by alexgarel — APPROVED** [review]
LGTM

**Pull request merged**